app/templates/buyers/delete_brief.html

```diff
@@ -0,0 +1,71 @@
+{% extends "_base_page.html" %}
+
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block pageTitle %}
+  {{ brief.title or brief.lotName }} – Digital Marketplace
+{% endblock %}
+
+{% block breadcrumb %}
+  {{ govukBreadcrumbs({
+    "items": [
+        {
+          "href": "/",
+          "text": "Digital Marketplace"
+      },
+      {
+          "href": url_for("buyers.buyer_dashboard"),
+          "text": "Your account"
+      },
+      {
+          "href": url_for("buyers.buyer_dos_requirements"),
+          "text": "Your requirements"
+      },
+      {
+        "href": url_for(
+            ".view_brief_overview",
+            framework_slug=brief.framework.slug,
+            lot_slug=brief['lotSlug'],
+            brief_id=brief['id']),
+        "text": brief['title']
+      },
+      {
+          "text": "Delete requirements?"
+      }
+    ]
+  }) }}
+{% endblock %}
+
+{% block mainContent %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <span class="govuk-caption-xl">{{ brief.get('title', brief['lotName']) }}</span>
+        <h1 class="govuk-heading-xl">Are you sure you want to delete these requirements?</h1>
+
+        {{ govukWarningText({
+        "text": "This action is final and cannot be undone.",
+        "iconFallbackText": "Warning",
+        "classes": "govuk-!-margin-bottom-8"
+        }) }}
+
+        <form method="post" action="{{ url_for('buyers.delete_a_brief', framework_slug=framework.slug, lot_slug=brief.lotSlug, brief_id=brief.id) }}" novalidate>
+            <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+            
+            {{ govukButton({
+                "text": "Yes, delete",
+                "classes": "govuk-button--warning govuk-!-margin-right-3",
+                "name": "delete_confirmed",
+                "value": True
+            }) }}
+
+            {{ govukButton({
+                "text": "Cancel",
+                "classes": "govuk-button--secondary",
+                "name": "cancel",
+                "value": True
+            }) }}
```

Comment:
Is there an advantage to having the buttons have values? We could instead make it so that the cancel button is actually a link (by using the href element to go back to the previous page), and then a plain POST to /delete will still work.

What are your thoughts?

Reply:
Good plan!